Description: Disable doctests which back depend on parent package time
Last-Update: 2023-03-02
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: b/Cargo.toml
===================================================================
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -42,6 +42,7 @@
 [lib]
 name = "time_core"
 path = "src/lib.rs"
+doctest = false
 
 [dependencies]
 
